[TH] Add Rules (#4410)
ppy/osu-wiki: wiki/Rules/th.md

> # กฏ

Reviewer: ```suggestion
# กฎ
```

> กฏเหล่านี้คือรากฐานที่ทำให้ชุมชน osu! ของเรานั้นเป็นที่ที่สนุก และต้อนรับทุก ๆ คน

Reviewer: ```suggestion
กฎเหล่านี้คือรากฐานที่ทำให้ชุมชน osu! ของเรานั้นเป็นที่ที่สนุก และต้อนรับทุก ๆ คน
```

> ## กฏของชุมชน

Reviewer: ```suggestion
## กฎของชุมชน
```

1. **ผู้เล่นหนึ่งคนมีบัญชีได้เพียงแค่ *หนึ่ง* บัญชีเท่านั้น ตลอดไป** บัญชี*อันแรก* ที่คุณสร้างขึ้นระหว่างการสมัครนั้นเป็นบัญชีอันเดียวที่คุณจะมีได้ จากขั้นตอนการเริ่มสร้างจนจบ บัญชีนี้คือ **คุณ** และไม่ใช่ของคนอื่น ไม่ใช่ของพี่ ไม่ใช่ของแม่ ของน้อง หรือของเพื่อน มันคือของ**คุณ** ห้ามให้บัญชีนี้กับใครเด็ดขาด และคุณไม่สามารถสร้างอันใหม่ได้ถ้าอันเก่าหายไป รักษามันไว้ให้ดีหละ

> 2. **อย่าเล่นขี้โกง** การใช้โปรแกรมภายนอกเพื่อที่จะได้ความได้เปรียบกว่าผู้เล่นคนอื่น ซึ่งรวมถึงการใช้ มาโคร (Macro) โปรแกรมช่วยเล็ง การปรับความเร็ว และอย่างอื่นอีกมากมาย ถ้าโปรแกรมนั้นช่วยเล่นเกมในส่วนที่คุณควรจะทำเอง มันไม่โอเคเลย

Reviewer: ```suggestion
2. **อย่าเล่นขี้โกง** การใช้โปรแกรมภายนอกเพื่อที่จะได้ความได้เปรียบกว่าผู้เล่นคนอื่น ซึ่งรวมถึงการใช้มาโคร (Macro) โปรแกรมช่วยเล็ง การปรับความเร็ว และอย่างอื่นอีกมากมาย ถ้าโปรแกรมนั้นช่วยเล่นเกมในส่วนที่คุณควรจะทำเอง มันไม่โอเคเลย
```

> 2. **อย่าเล่นขี้โกง** การใช้โปรแกรมภายนอกเพื่อที่จะได้ความได้เปรียบกว่าผู้เล่นคนอื่น ซึ่งรวมถึงการใช้ มาโคร (Macro), โปรแกรมช่วยเล็ง, การปรับความเร็ว และอย่างอื่นอีกมากมาย ถ้าโปรแกรมนั้นช่วยเล่นเกมในส่วนที่คุณควรจะทำ เช่นการกด, การเล็ง เป็นต้น มันไม่โอเคเลย

Reviewer: ```suggestion
2. **อย่าเล่นขี้โกง** การใช้โปรแกรมภายนอกเพื่อที่จะได้ความได้เปรียบกว่าผู้เล่นคนอื่น ซึ่งรวมถึงการใช้ มาโคร (Macro), โปรแกรมช่วยเล็ง, การปรับความเร็ว และอื่น ๆ อีกมากมาย ถ้าโปรแกรมนั้นช่วยเล่นเกมในส่วนที่คุณควรจะทำเอง มันไม่โอเคเลย
```
ส่วนข้างหลังเราคิดว่าไม่จำเป็นนะถ้าแปลตรงตัวแบบ translate literally จาก
> If a program is doing something to help you play the game that you should be doing yourself, it isn't okay!

> 3. **ทำดีต่อกันและกัน** การคุกคามหรือการทะเลาะกัน ไม่ควรนำมาอยู่ในชุมชนของ osu! เรามาที่นี้เพื่อที่จะมานั่งกดวงกลม ตีกลอง รับผลไม้ที่ตกลงมา และกดปุ่มได้มากถึง 18 ปุ่มภายในครั้งเดียว! ไม่ได้มาเพื่อทะเลาะกัน

Reviewer: ```suggestion
3. **ทำดีต่อกันและกัน** การคุกคามหรือการทะเลาะกัน ไม่ควรนำมาอยู่ในชุมชนของ osu! เรามาที่นี่เพื่อที่จะมานั่งกดวงกลม ตีกลอง รับผลไม้ที่ตกลงมา และกดปุ่มได้มากถึง 18 ปุ่มภายในครั้งเดียว! ไม่ได้มาเพื่อทะเลาะกัน
```
ที่นี้ -> ที่นี่

> 3. **ทำดีต่อกันและกัน** การคุกคามหรือการทะเลาะกัน ไม่ควรนำมาอยู่ในชุมชนของ osu! เรามาที่นี้เพื่อที่จะมานั่งกดวงกลม, ตีกลอง, รับผลไม้ที่ตกลงมา, กดปุ่มมากได้ถึง 18ปุ่มภายในครั้งเดียว! ไม่ได้มาเพื่อทะเลาะกัน

Reviewer: ```suggestion
3. **ทำดีต่อกันและกัน** การคุกคามหรือการทะเลาะกัน ไม่ควรนำมาอยู่ในชุมชนของ osu! เรามาที่นี้เพื่อที่จะมานั่งกดวงกลม, ตีกลอง, รับผลไม้ที่ตกลงมา, และกดปุ่มได้มากถึง 18 ปุ่มภายในครั้งเดียว! ไม่ได้มาเพื่อทะเลาะกัน
```
fix spacing / wording

> 4. **อย่าทำตัวเป็นพวกน่ารำคาญ** ถ้ามีตอนไหนที่คุณคิดว่าตัวเองได้ทำผิดกฏข้อนี้ไปหรือเปล่า คุณน่าจะทำแหละ

Reviewer: ```suggestion
4. **อย่าทำตัวเป็นพวกน่ารำคาญ** ถ้ามีตอนไหนที่คุณคิดว่าตัวเองได้ทำผิดกฎข้อนี้ไปหรือเปล่า คุณน่าจะทำแหละ
```

> 5. **เราเป็นชุมชนที่มีทุกวัย** ดังนั้นแปลว่า อะไรก็ตามที่เป็น 18+/[NSFW](https://en.wikipedia.org/wiki/Not_safe_for_work) เช่นเรื่องการเสพยา หรือเรื่องหื่นกามต่าง ๆ จะไม่ถูกต้อนรับที่นี่ จะมีข้อยกเว้นเฉพาะในเรื่องที่เหมาะสมเท่านั้น แต่ถ้าตามปกติแล้ว อย่าพูดถึงเรื่องพวกนี้

Reviewer: ```suggestion
5. **เราเป็นชุมชนที่มีทุกวัย** ดังนั้นแปลว่า อะไรก็ตามที่เป็น 18+/[NSFW](https://en.wikipedia.org/wiki/Not_safe_for_work) เช่นเรื่องการเสพยา หรือเรื่องหื่นกามต่าง ๆ จะไม่ถูกต้อนรับที่นี่ จะมีข้อยกเว้นเฉพาะในเรื่องที่เหมาะสม แต่ถ้าตามปกติแล้ว อย่าพูดถึงเรื่องพวกนี้
```
contradicatory นิดนึงอ่ะ อันนี้ดีกว่ามั้ยๆ

> 6. **ถ้าเรื่องอะไรอยู่นอกเหนือกฏ ควรใช้ความคิดเบื้องต้นคิดเอง** ทีมงานมีสิทธิ์ที่จะดูแลลงโทษผู้ใช้ตามความเหมาะสม

Reviewer: ```suggestion
6. **ถ้าเรื่องอะไรอยู่นอกเหนือกฎ ควรใช้ความคิดเบื้องต้นคิดเอง** ทีมงานมีสิทธิ์ที่จะดูแลลงโทษผู้ใช้ตามความเหมาะสม
```

> ## กฏของแชทในเกม

Reviewer: ```suggestion
## กฎของแชทในเกม
```

> 6. **ถ้าเรื่องอะไรอยู่นอกเหนือกฏ ควรใช้ความคิดเบื้องต้นคิดเอง** ทีมงานดูแลลงโทษผู้ใช้ตามความเหมาะสม

Reviewer: ```suggestion
6. **ถ้าเรื่องอะไรอยู่นอกเหนือกฏ ควรใช้ความคิดเบื้องต้นคิดเอง** ทีมงานมีสิทธิ์ที่จะดูแลลงโทษผู้ใช้ตามความเหมาะสม
```
"has explicit discretion"

1. **ให้เกียรติซึ่งกันและกัน หลีกเลี่ยงการฟลัด และสแปม** การที่คนคนเดียวใช้แชทไปครึ่งจอภายในครั้งเดียวนั้นมันน่ารำคาญ ดังนั้นหลีกเลี่ยงการทำอย่างงั้นถ้าเป็นไปได้ คนอื่นก็อยากจะพูดคุยเหมือนกัน!
2. **ปล่อยให้ผู้ดูแลจัดการปัญหาภายในแชท** คุณสามารถรายงานผู้ใช้คนอื่นได้โดยการใช้ [คำสั่ง `!report`](/wiki/Reporting_Bad_Behaviour) และพวกเขาจะจัดการที่เหลือเอง

> 3. **ถ้าใครโดน Timed out โดนเงียบ (Silenced) หรือโดนแบน (Restricted) ให้ปล่อยไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบนกับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!

Reviewer: ```suggestion
3. **ถ้าใครโดน timed out โดนเงียบ (silenced) หรือโดนแบน (restricted) ให้ปล่อยไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบนกับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!
```
Make capitalization consistent

> 3. **ถ้าใครโดน Timed out, โดนเงียบ (Silenced), หรือโดนแบน (Restricted) ปล่อยมันไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบน กับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!

Reviewer: ```suggestion
3. **ถ้าใครโดน Timed out, โดนเงียบ (Silenced), หรือโดนแบน (Restricted) ให้ปล่อยไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบน กับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!
```
ตอนนี้มันเหมือนแบบคำว่า "มัน" พูดถึงคนที่โดน silence แบบหยาบอ่ะ อันนี้ดีกว่ามั้ยย

Author: ตอนแปล สมองเบลอๆ คิดคำไม่ค่อยออก 555 ใช่ฮะมาอ่านซ้ำก็ว่าแปลกแหละ

> 3. **ถ้าใครโดน Timed out, โดนเงียบ (Silenced), หรือโดนแบน (Restricted) ปล่อยมันไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบน กับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!

Reviewer: ```suggestion
3. **ถ้าใครโดน Timed out, โดนเงียบ (Silenced), หรือโดนแบน (Restricted) ปล่อยมันไปและอย่าไปพูดถึงเรื่องนั้น** ถ้าใครโดนแบนนั้น เรื่องนั้นเกี่ยวกับคนที่โดนแบนกับผู้ดูแลเท่านั้น ไม่ใช่เรื่องของคนอื่นเลย!
```
ไม่วรรคมั้ย

4. **สนับสนุนการเล่นไม่โกง และอย่าไปให้ความสนใจกับพวกโกง หรือพวก toxic** น่าเสียดายที่มันมีคนพวกนี้อยู่ และการให้ความสนใจพวกเขานั้น จะทำให้เขาอยากทำต่อไป หรือชักชวนผู้อื่นให้ทำด้วย คุณสามารถรายงานได้โดยการใช้คำสั่ง `!report` หรือปุ่มรายงานบนหน้าโปรไฟล์ของพวกเขา และไม่ต้องไปสนใจเขา

> 5. **ทำให้แชทสะอาด และเหมาะสมสำหรับสังคมทุกวัย หลีกเลี่ยงเนื้อหา NSFW ถ้าเป็นไปได้** ของพวก NSFW มันไม่ค่อยเหมาะกับชุมชนที่มีทุกวัย ทำอะไรก็ได้ที่คุณอยากทำในข้อความส่วนตัว (อย่างมีเหตุผล) แต่ทำให้แชทส่วนรวมสะอาดปลอดภัยจากเรื่อง NSFW สำหรับทุกคน

Reviewer: ```suggestion
5. **ทำให้แชทสะอาด และเหมาะสมสำหรับสังคมทุกวัย หลีกเลี่ยงเนื้อหา NSFW ถ้าเป็นไปได้** เนื้อหา NSFW ไม่ค่อยเหมาะกับชุมชนที่มีทุกวัย คุณสามารถทำอะไรก็ได้ที่คุณอยากทำในข้อความส่วนตัว (อย่างมีเหตุผล) แต่ทำให้แชทส่วนรวมสะอาดปลอดภัยจากเรื่อง NSFW สำหรับทุกคน
```
Improve wording

6. **คิดก่อนจะส่งอะไรให้คนอื่น โดยเฉพาะสิ่งที่มีเนื้อหารบกวนใจ หรือสิ่งที่ทำคนอื่นไม่สบายใจ** ถ้าคุณต้องคิดซ้ำว่า สิ่งที่คุณจะส่งนั้นมันเหมาะสมไหม มันคงจะไม่เหมาะสมแหละ หลีกเลี่ยงสิ่งที่ เร็ว ไฟกระพริบ (บางคนมีโรคเกี่ยวกับพวกนี้) เลือด ภาพที่แสดงถึงการละเมิดต่าง ๆ

> 7. **ส่งเฉพาะลิงก์ที่เกี่ยวข้อง และเว็บไซต์ที่ถูกกฏหมาย** เว็บไซต์ของเถื่อนต่าง ๆ (อันที่ถูกกฏหมายอย่าง Crunchyroll หรือ Netflix นั้นโอเค) อย่างอื่นที่เกี่ยวกับ "โปรแกรมเถื่อน" ต่าง ๆ นั้นไม่โอเคสักเท่าไหร่ และเราไม่ต้องการมันในแชทของเรา การโฆษณา Twitch สำหรับเกมอื่นนั้นโอเค แต่เฉพาะในช่อง `#videogames` เท่านั้น <!-- I changed AnimeLab to Netflix since Netflix are more well known in Thailand -->

Reviewer: ```suggestion
7. **ส่งเฉพาะลิงก์ที่เกี่ยวข้อง และเว็บไซต์ที่ถูกกฎหมาย** เว็บไซต์ของเถื่อนต่าง ๆ (อันที่ถูกกฎหมายอย่าง Crunchyroll หรือ Netflix นั้นโอเค) อย่างอื่นที่เกี่ยวกับ "โปรแกรมเถื่อน" ต่าง ๆ นั้นไม่โอเคสักเท่าไหร่ และเราไม่ต้องการมันในแชทของเรา การโฆษณา Twitch สำหรับเกมอื่นนั้นโอเค แต่เฉพาะในช่อง `#videogames` เท่านั้น <!-- I changed AnimeLab to Netflix since Netflix are more well known in Thailand -->
```

> 7. **ส่งเฉพาะลิงก์ที่เกี่ยวข้อง และเว็บไซต์ที่ถูกกฏหมาย** เว็บไซต์ของเถื่อนต่าง ๆ (อันที่ถูกกฏหมายอย่าง Crunchyroll หรือ Netflix นั้นโอเค) อย่างอื่นที่เกี่ยวกับ "โปรแกรมเถื่อน" ต่าง ๆ นั้นไม่ค่อยจะโอเคสักเท่าไหร่ และเราไม่ต้องการมันในแชทของเรา การโฆษณา Twitch สำหรับเกมอื่นนั้นโอเค แต่เฉพาะในช่อง `#videogames` เท่านั้น <!-- I changed AnimeLab to Netflix since Netflix are more well known in Thailand -->

Reviewer: ```suggestion
7. **ส่งเฉพาะลิงก์ที่เกี่ยวข้อง และเว็บไซต์ที่ถูกกฏหมาย** เว็บไซต์ของเถื่อนต่าง ๆ (อันที่ถูกกฏหมายอย่าง Crunchyroll หรือ Netflix นั้นโอเค) อย่างอื่นที่เกี่ยวกับ "โปรแกรมเถื่อน" ต่าง ๆ นั้นไม่โอเคสักเท่าไหร่ และเราไม่ต้องการมันในแชทของเรา การโฆษณา Twitch สำหรับเกมอื่นนั้นโอเค แต่เฉพาะในช่อง `#videogames` เท่านั้น <!-- I changed AnimeLab to Netflix since Netflix are more well known in Thailand -->
```
จากต้นฉบับ "aren't okay" แบบว่า clear cut อ่ะ เปลี่ยนเป็น "ไม่โอเคเลย" ก็ได้นะ

8. **อย่าโฆษณาโปรแกรมพูดคุยภายนอก รวมถึง เซิฟเวอร์/กลุ่ม Discord Teamspeak หรือ Skype** แชท osu! นั้นก็มีไว้สำหรับ osu! โดยเฉพาะ มันไม่ได้มีไว้สำหรับการส่งลิงก์เข้าเซิฟเวอร์ส่วนตัวของคุณ ส่งมันในข้อความส่วนตัว (PM) ถ้ามันจำเป็นจริง ๆ เท่านั้น

> 9. **ในช่องแชทที่มีภาษาระบุ คุณต้องพูดภาษานั้น (เช่น พูดอังกฤษใน `#english` พูดไทยใน `#thai`) ถ้าคุณกำลังมีส่วนร่วมในการสนทนา** ในช่องที่ไม่ได้มีภาษาบอกไว้ มันคือช่องที่ต้องพูดอังกฤษเท่านั้น แต่ในช่อง `#lobby` และ `#help` นั้นเป็นช่องที่พูดได้ทุกภาษาไม่เหมือนช่องอื่น ๆ *[ผู้ดูแล](/wiki/People/The_Team/Global_Moderation_Team) ไม่จำเป็นจะต้องทำตามกฏข้อนี้ระหว่างปฏิบัติหน้าที่*

Reviewer: ```suggestion
9. **ในช่องแชทที่มีภาษาระบุ คุณต้องพูดภาษานั้น (เช่น พูดอังกฤษใน `#english` พูดไทยใน `#thai`) ถ้าคุณกำลังมีส่วนร่วมในการสนทนา** ในช่องที่ไม่ได้มีภาษาบอกไว้ มันคือช่องที่ต้องพูดอังกฤษเท่านั้น แต่ในช่อง `#lobby` และ `#help` นั้นเป็นช่องที่พูดได้ทุกภาษาไม่เหมือนช่องอื่น ๆ *[ผู้ดูแล](/wiki/People/The_Team/Global_Moderation_Team) ไม่จำเป็นจะต้องทำตามกฎข้อนี้ระหว่างปฏิบัติหน้าที่*
```

> ## กฏของฟอรั่ม

Reviewer: ```suggestion
## กฎของฟอรั่ม
```

> 9. **ในช่องแชทที่มีภาษาระบุ คุณต้องพูดภาษานั้น (เช่น พูดอังกฤษใน `#english`, พูดไทยใน `#thai`) ถ้าคุณกำลังมีส่วนร่วมในการสนทนา** ในช่องที่ไม่ได้มีภาษาบอกไว้ มันคือช่องที่ต้องพูดอังกฤษเท่านั้น แต่ในช่อง `#lobby` และ `#help` นั้นเป็นช่องที่พูดได้ทุกภาษาไม่เหมือนช่องอื่น ๆ *ทีมงานของ [ผู้ดูแล](/wiki/People/The_Team/Global_Moderation_Team) ไม่จำเป็นจะต้องทำตามกฏข้อนี้ระหว่างปฏิบัติหน้าที่*

Reviewer: ```suggestion
9. **ในช่องแชทที่มีภาษาระบุ คุณต้องพูดภาษานั้น (เช่น พูดอังกฤษใน `#english`, พูดไทยใน `#thai`) ถ้าคุณกำลังมีส่วนร่วมในการสนทนา** ในช่องที่ไม่ได้มีภาษาบอกไว้ มันคือช่องที่ต้องพูดอังกฤษเท่านั้น แต่ในช่อง `#lobby` และ `#help` นั้นเป็นช่องที่พูดได้ทุกภาษาไม่เหมือนช่องอื่น ๆ *ทีมงาน[ผู้ดูแล](/wiki/People/The_Team/Global_Moderation_Team) ไม่จำเป็นจะต้องทำตามกฏข้อนี้ระหว่างปฏิบัติหน้าที่*
```
ไม่วรรค

Author: I think removing the whole "ทีมงาน" thing might be better

> 1. **โพสต์เฉพาะสิ่งที่มีประโยชน์, น่าสนใจ, หรือหัวข้อที่ดึงดูดความสนใจ** ฟอรั่มนั้นมีไว้สำหรับการสนทนากัน ดังนั้นอย่าทำอะไรชุ่ย ๆ ที่ทำให้เสียเวลาคนอื่น

Reviewer: ```suggestion
1. **โพสต์เฉพาะสิ่งที่มีประโยชน์, น่าสนใจ, หรือหัวข้อที่น่ามีส่วนร่วม** ฟอรั่มนั้นมีไว้สำหรับการสนทนากัน ดังนั้นอย่าทำอะไรชุ่ย ๆ ที่ทำให้เสียเวลาคนอื่น
```
ดึงดูดความสนใจมันประมาณแบบ attention seeking อ่ะ ถ้าเป็น​ "น่ามีส่วนร่วม" จะดีกว่ามั้ย

2. **ทำให้โพสต์และการตอบกลับโพสต์ของคุณเหมาะสมกับทุกเพศทุกวัย** เรื่องพวก NSFW นั้นไม่เหมาะกับชุมชนที่มีหลากหลายวัย แต่ว่ามันก็ไม่มีอะไรจะหยุดคุณในการคุยเรื่องพวกนี้ในแชทส่วนตัวนะ

> 3. **ให้คำติชมที่สมเหตุสมผล โดยที่ไม่ใช้อารมณ์ส่วนตัวเข้ามาเกี่ยว** การติชมเป็นสิ่งที่สุดยอดไปเลยถ้าทำได้อย่างถูกต้อง แต่ถ้าคุณจะต้องการโจมตีเป็นการส่วนตัวเพื่อที่จะแสดงว่าตัวเองถูก คุณควรรู้ไว้ว่ามันเป็นการติชมที่ผิด และคุณควรรู้สึกแย่

Reviewer: ```suggestion
3. **ให้คำติชมที่สมเหตุสมผล โดยที่ไม่ใช้อารมณ์ส่วนตัวเข้ามาเกี่ยว** การติชมเป็นสิ่งที่สุดยอดไปเลยถ้าทำได้อย่างถูกต้อง แต่ถ้าคุณโจมตีเป็นการส่วนตัวเพื่อที่จะแสดงว่าตัวเองถูก มันเป็นการติชมที่ผิด และคุณควรรู้สึกแย่
```
improve translation

> 4. **อย่าดันกระทู้ (threads) ที่เก่าแล้วเข้ามา ถ้าการสนทนามันเงียบไปนานแล้ว** ถ้าหัวข้อไหนนั้นมันเงียบไปเป็นปีแล้ว มันก็คงจะไม่เข้ายุคแล้วหละ ดังนั้นปล่อยให้มันหายไป โพสต์อันใหม่, หรือหาว่ามีใครตอบคำถามคุณหรือยัง

Reviewer: ```suggestion
4. **อย่าดันกระทู้ (threads) ที่เก่าแล้วเข้ามา ถ้าการสนทนามันเงียบไปนานแล้ว** ถ้าหัวข้อไหนนั้นมันเงียบไปเป็นปีแล้ว มันก็คงจะไม่เข้ายุคแล้วหละ ดังนั้นปล่อยให้มันหายไป โพสต์อันใหม่ หรือค้นหาว่ามีใครตอบคำถามคุณหรือยัง
```
คิดว่าไม่ควรใช้ comma นะในภาษาไทย

5. **ทำให้ลายเซ็น และ รูปโปรไฟล์ของคุณสมเหตุสมผล** ถ้ามันเคลื่อนไหวได้ มันไม่ควรจะกระพริบมากจนเกินไป และภาพก็ควรจะเล็กพอที่จะทำให้คนที่มีการเชื่อมต่อที่ช้าเข้าถึงได้ ที่ดีที่สุดคือต่ำกว่า 500 กิโลไบต์ (500 KB) ถ้าเป็นไปได้

> 6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** พวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลด หรือเล่นเนื้อหา โดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฏข้อนี้ด้วย

Reviewer: ```suggestion
6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** พวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลดหรือเล่นเนื้อหาโดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฏข้อนี้ด้วย
```
Spacing issue

> 6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** พวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลดหรือเล่นเนื้อหาโดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฏข้อนี้ด้วย

Reviewer: ```suggestion
6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** พวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลดหรือเล่นเนื้อหาโดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฎข้อนี้ด้วย
```

> 6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** ของพวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลด หรือเล่นเนื้อหา โดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฏข้อนี้ด้วย

Reviewer: ```suggestion
6. **อย่าโพสต์โฆษณา ลิงก์ขายของ หรืออะไรก็ตามที่น่ารำคาญ หรือล่วงล้ำผู้อื่น** พวกลิงก์ขายของ นั้นมันดูน่าสนุกสำหรับคนโพสต์มัน แต่มันน่ารำคาญสำหรับคนอื่น ดังนั้นอย่างทำมัน การโหลด หรือเล่นเนื้อหา โดยอัตโนมัติ (Autoloading/Autoplaying) อยู่ในกฏข้อนี้ด้วย
```
repetitive

> 7. **เราไม่อนุญาต การลิงก์ไปแอพพลิเคชั่นหรือเครื่องมืออื่น ๆ ทีมีจุดมุ่งหมายไม่ชัดเจน** เอาง่าย ๆ เลย ถ้าคิดว่าเครื่องมือที่กล่าวมาผิดกฏหมายหรือไม่ เราไม่อนุญาตให้โพสต์สิ่งเหล่านั้น

Reviewer: ```suggestion
7. **เราไม่อนุญาต การลิงก์ไปแอพพลิเคชั่นหรือเครื่องมืออื่น ๆ ทีมีจุดมุ่งหมายไม่ชัดเจน** เอาง่าย ๆ เลย ถ้าคิดว่าเครื่องมือที่กล่าวมาผิดกฎหมายหรือไม่ เราไม่อนุญาตให้โพสต์สิ่งเหล่านั้น
```

> ## กฏการอัปโหลดบีทแมพ

Reviewer: ```suggestion
## กฎการอัปโหลดบีทแมพ
```

> 7. **เราไม่อนุญาติ การลิงก์ไปแอพพลิเคชั่นหรือเครื่องมืออื่น ๆ ทีมีจุดมุ่งหมายไม่ชัดเจน** เอาง่าย ๆ เลย ถ้าคิดว่าเครื่องมือที่กล่าวมาผิดกฏหมายหรือไม่ เราไม่อนุญาติให้โพสต์สิ่งเหล่านั้น

Reviewer: ```suggestion
7. **เราไม่อนุญาต การลิงก์ไปแอพพลิเคชั่นหรือเครื่องมืออื่น ๆ ทีมีจุดมุ่งหมายไม่ชัดเจน** เอาง่าย ๆ เลย ถ้าคิดว่าเครื่องมือที่กล่าวมาผิดกฏหมายหรือไม่ เราไม่อนุญาตให้โพสต์สิ่งเหล่านั้น
```
อนุญาตจ้า ไม่ใช้อนุญาติ

> *โน้ต: กฏพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)*

Reviewer: ```suggestion
*หมายเหตุ: กฏพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)*
```
เพิ่งเห็นครับ เปลี่ยนเป็นคำแปลดีมั้ย ไม่ทับศัพท์

Reviewer: ![image](https://user-images.githubusercontent.com/20501245/98430111-e3676f00-20dd-11eb-8a70-c84f06d2a642.png)
ลองหาดูในไฟล์แล้วครับ เจอที่ใช้คำว่า "หมายเหตุ" คำเดียวในบทความ GMT ถ้าเปลี่ยนก็คงจะต้องเปลี่ยนอย่างอื่นอีก เอาตามเห็นสมควรเลยครับ

Author: บทความ GMT ผมไม่ได้แปลอ่ะสิ แต่ผมว่าทุกอันที่ผมแปลน่าจะเป็น โน้ต นะ
ผมว่าใช้โน้ตน่าจะดีกว่าแหละฮะ

Author: Actually nvm เอา หมายเหตุนั้นแหละ
เดี๋ยวไปเปลี่ยนใน article อื่นใน #4415 เอา

Reviewer: ใช่ คิดว่าถ้าใช้คำว่าโน้ตมันแปลกๆ นิดนึง สร้าง PR ใหม่ที่ไม่ใช่อันนั้นเลยดีมั้ย เพราะว่าดูมีหลายไฟล์อยู่

Author: ผมว่า clear PR ทั้งหมดตอนนี้ก่อนแล้วค่อย PR ใหม่ กลัวแก้ไปแล้ว conflict กับ PR ตอนนี้

> *หมายเหตุ: กฏพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)*

Reviewer: ```suggestion
*หมายเหตุ: กฎพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)*
```

> *โน้ต: กฏพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่ แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)

Reviewer: ```suggestion
*โน้ต: กฏพวกนี้มีผลต่อ **บีทแมพที่ถูกอัปโหลดทั้งหมด** ไม่ใช่แค่พวกที่ถูก ranked สำหรับกฏเกณฑ์ที่เกี่ยวกับบีทแมพ ranked ดูได้ที่ [เกณฑ์การอนุมัติบีทแมพ](/wiki/Ranking_Criteria)
```

> กฏพวกนี้มีผลต่อบีทแมพที่ถูกอัปโหลดผ่าน [Beatmap Submission System](/wiki/Glossary#bss) ภายในเกม โดยกระทู้/การสนทนา จะถูกสร้างขึ้นด้วย osu! client ในระหว่างที่บีทแมพถูกอัปโหลดผ่าน [เครื่องมือแก้ไขบีทแมพ](/wiki/Beatmap_Editor)

Reviewer: ```suggestion
กฎพวกนี้มีผลต่อบีทแมพที่ถูกอัปโหลดผ่าน [Beatmap Submission System](/wiki/Glossary#bss) ภายในเกม โดยกระทู้/การสนทนา จะถูกสร้างขึ้นด้วย osu! client ในระหว่างที่บีทแมพถูกอัปโหลดผ่าน [เครื่องมือแก้ไขบีทแมพ](/wiki/Beatmap_Editor)
```

> กฏพวกนี้มีผลต่อบีทแมพที่ถูกอัปโหลดผ่าน [Beatmap Submission System](/wiki/Glossary#bss) ภายในเกม กระทู้/การสนทนา จะถูกสร้างขึ้นด้วย osu! client ในระหว่างที่บีทแมพถูกอัปโหลดผ่าน [เครื่องมือแก้ไขบีทแมพ](/wiki/Beatmap_Editor)

Reviewer: ```suggestion
กฏพวกนี้มีผลต่อบีทแมพที่ถูกอัปโหลดผ่าน [Beatmap Submission System](/wiki/Glossary#bss) ภายในเกม โดยกระทู้/การสนทนา จะถูกสร้างขึ้นด้วย osu! client ในระหว่างที่บีทแมพถูกอัปโหลดผ่าน [เครื่องมือแก้ไขบีทแมพ](/wiki/Beatmap_Editor)
```
รู้สึกคำมันติดกันเกินไปอ่ะ

> 1. **เช็คให้แน่ใจว่าคุณมีสิทธิในการใช้อะไรก็ตามที่อยู่ในบีทแมพของคุณ** นี่รวมถึง เพลง, วีดีโอ, hitsounds, ภาพต่าง ๆ , หรือเนื้อหาอะไรก็ตามที่ไม่ใช่ของคุณเอง ถ้าคุณไม่แน่ใจว่าหาเนื้อหาที่ไหนที่ไม่ต้องเสียตังค์ในการใช้ ลองไปเช็คที่คลัง [Featured Artist](https://osu.ppy.sh/beatmaps/artists) ของเราสิ ที่นั้นมีเพลงที่จดสิขสิทธิ์แล้วสำหรับการใช้ใน osu! 100% และเรามี [หลักเกณฑ์การใช้งานเนื้อหา](Content_Usage_Guidelines)

Reviewer: ```suggestion
1. **เช็คให้แน่ใจว่าคุณมีสิทธิในการใช้อะไรก็ตามที่อยู่ในบีทแมพของคุณ** นี่รวมถึง เพลง, วีดีโอ, hitsounds, ภาพต่าง ๆ , หรือเนื้อหาอะไรก็ตามที่ไม่ใช่ของคุณเอง ถ้าคุณไม่แน่ใจว่าหาเนื้อหาที่ไหนที่ไม่ต้องเสียตังค์ในการใช้ ลองไปเช็คที่คลัง [Featured Artist](https://osu.ppy.sh/beatmaps/artists) ของเราสิ ที่นั้นมีเพลงที่จดสิขสิทธิ์แล้วสำหรับการใช้ใน osu! 100% และดูข้อมูลเพิ่มเติมได้ที่ [หลักเกณฑ์การใช้งานเนื้อหา](Content_Usage_Guidelines)
```

Author: Changed to "คุณสามารถดูข้อมูลเพิ่มเติมได้ที่"

2. **อย่าออกนอกเรื่องในการสนทนาบีทแมพ** การสนทนาพวกนี้นั้นจะเกี่ยวกับบีทแมพที่พูดถึงอยู่ และไม่เกี่ยวกับอย่างอื่นเลย ถ้าคุณมีปัญหาอะไรที่ไม่เกี่ยวกับบีทแมพนั้น ๆ โดยตรง ให้โพสต์ในฟอรั่มที่ถูกต้อง

> 3. **อย่าลอกเลียน หรือพยายามขโมยผลงานของผู้อื่น** อย่าอัปโหลดผลงานของผู้อื่นโดยไม่ได้รับอนุญาติ (รวมถึง แต่ไม่จำกัดเพียง สกิน Guest difficulties)

Reviewer: ```suggestion
3. **อย่าลอกเลียน หรือพยายามขโมยผลงานของผู้อื่น** อย่าอัปโหลดผลงานของผู้อื่นโดยไม่ได้รับอนุญาติ (รวมถึง แต่ไม่จำกัดเพียง สกิน หรือ guest difficulties)
```

4. **ทำตาม [กฎของเนื้อหาเพลง](Song_Content_Rules) และ [การพิจารณาเนื้อหาภาพ](Visual_Content_Considerations)**

> ## จะเกิดอะไรขึ้นถ้าฉันทำผิดกฏ?

Reviewer: ```suggestion
## จะเกิดอะไรขึ้นถ้าฉันทำผิดกฎ?
```

> 4. **ทำตาม [กฎของเนื้อหาเพลง](Song_Content_Rules) and [การพิจารณาเนื้อหาภาพ](Visual_Content_Considerations).**

Reviewer: ```suggestion
4. **ทำตาม [กฎของเนื้อหาเพลง](Song_Content_Rules) และ [การพิจารณาเนื้อหาภาพ](Visual_Content_Considerations).**
```

Author: เบลอจัดๆ

> ถ้าคุณนั้นทำผิดกฏของชุมชน สิ่งแย่ ๆ จะเกิดขึ้น

Reviewer: ```suggestion
ถ้าคุณนั้นทำผิดกฎของชุมชน สิ่งแย่ ๆ จะเกิดขึ้น
```

สิ่งแย่ ๆ พวกนี้รวมถึง (*แต่ไม่จำกัดเพียง*):

> - การเสียสิทธิในการพูดในแชทเกมออกระยะนึง (เรียกว่า เงียบ (Silence))

Reviewer: ```suggestion
- การเสียสิทธิในการพูดในแชทเกมในระยะนึง (เรียกว่า เงียบ (silence))
```

> - การเอาเนื้อหาที่ผิดกฏออกจาก userpage หน้าโปรไฟล์ บีทแมพ

Reviewer: ```suggestion
- การเอาเนื้อหาที่ผิดกฏออกจาก userpage หน้าโปรไฟล์ หรือบีทแมพ
```

> - การเอาเนื้อหาที่ผิดกฏออกจาก userpage หน้าโปรไฟล์ หรือบีทแมพ

Reviewer: ```suggestion
- การเอาเนื้อหาที่ผิดกฎออกจาก userpage หน้าโปรไฟล์ หรือบีทแมพ
```

- การเสียสิทธิในการแก้ไข หรือแก้ userpage หรือข้อมูลโปรไฟล์

> - การเสียสิทธิในการเล่นหรือผ่านเข้ารอบในทัวร์นาเม้นต์ของทางการ

Reviewer: ```suggestion
- การเสียสิทธิในการเล่นหรือผ่านเข้ารอบในทัวร์นาเม้นต์ที่จัดขึ้นอย่างเป็นทางการ
```
Improve translation

> การเงียบหรือการลงโทษอื่น ๆ จะเพิ่มขึ้นความยาวนานเกือบทวีคูณ**อย่างอัตโนมัติ** ในทุก ๆ ครั้งที่คุณโดนลงโทษ ผู้ดูแลไม่สามารถควบคุมเวลาที่คุณโดนเงียบได้ คนที่ทำผิดบ่อย ๆ จะถูกเงียบนานขึ้นเรื่อย ๆ

Reviewer: ```suggestion
การเงียบหรือการลงโทษอื่น ๆ จะเพิ่มขึ้นความยาวนานเกือบทวีคูณ**อย่างอัตโนมัติ**ในทุก ๆ ครั้งที่คุณโดนลงโทษ ผู้ดูแลไม่สามารถควบคุมเวลาที่คุณโดนเงียบได้ คนที่ทำผิดบ่อย ๆ จะถูกเงียบนานขึ้นเรื่อย ๆ
```
เว้นวรรคให้มันดูแยกประโยคออก

> บางกรณีก็จะมีเวลาขั้นต่ำ (ปกติคือ 48 ชั่วโมง หรือมากกว่า) ถ้ามันเป็นกรณีร้ายแรงระดับนึง แต่กรณีพวกนี้มีไว้สำหรับกรณีร้ายแรงจริง ๆ เท่านั้น เช่น การแสดงภาพที่แสดงถึงความรุนแรง หรือคุกคามผู้ใช้คนอื่น ๆ แบบไม่สำนึกถูกผิด

Reviewer: ```suggestion
บางกรณีก็จะมีเวลาขั้นต่ำ (ปกติคือ 48 ชั่วโมง หรือมากกว่า) ถ้ามันเป็นกรณีร้ายแรงระดับนึง แต่กรณีพวกนี้มีไว้สำหรับกรณีร้ายแรงจริง ๆ เท่านั้น เช่น การส่งภาพที่แสดงถึงความรุนแรง หรือคุกคามผู้ใช้คนอื่น ๆ แบบไม่สำนึกถูกผิด
```
แสดง -> ส่ง

> บางกรณีก็จะมีเวลาขั้นต่ำ (ปกติคือ 48 ชั่วโมง หรือมากกว่า) ถ้ามันเป็นกรณีร้ายแรงระดับนึง แต่กรณีพวกนี้มีไว้สำหรับกรณีร้ายแรงจริง ๆ เท่านั้น เช่น การแสดงภาพที่แสดงถึงความรุนแรง หรือคุกคามผู้ใช้คนอื่น ๆ มากเกินไป

Reviewer: ```suggestion
บางกรณีก็จะมีเวลาขั้นต่ำ (ปกติคือ 48 ชั่วโมง หรือมากกว่า) ถ้ามันเป็นกรณีร้ายแรงระดับนึง แต่กรณีพวกนี้มีไว้สำหรับกรณีร้ายแรงจริง ๆ เท่านั้น เช่น การแสดงภาพที่แสดงถึงความรุนแรง หรือคุกคามผู้ใช้คนอื่น ๆ แบบไม่สำนึกถูกผิด
```
แปลคำว่า **unrepetant** ใหม่

> ถ้าทำโดนลงโทษไปแล้วหลายรอบ แต่คุณยังจะทำผิดกฏต่อ บัญชีของคุณจะถูกแบนถาวร (restricted)

Reviewer: ```suggestion
ถ้าทำโดนลงโทษไปแล้วหลายรอบ แต่คุณยังจะทำผิดกฎต่อ บัญชีของคุณจะถูกแบนถาวร (restricted)
```

> ถ้าทำโดนลงโทษไปแล้วหลายรอบ แต่คุณยังจะทำผิดกฏต่อ บัญชีของคุณจะถูก Restricted ถาวร (แบนถาวร)

Reviewer: ```suggestion
ถ้าทำโดนลงโทษไปแล้วหลายรอบ แต่คุณยังจะทำผิดกฏต่อ บัญชีของคุณจะถูกแบนถาวร (restricted)
```

> บัญชีที่ถูก Restricted จะไม่สามารถเล่นออนไลน์ได้ ทำคะแนนใหม่ พูดคุยกับผู้เล่นอื่น ซึ่งก็คือการ "แบน" การทำอะไรก็ตามที่เกี่ยวกับชุมชน การ restict นั้นสามารถนำออกได้โดยการ appeal กับทีมงานของ osu! หลังจาก 3 เดือนได้ผ่านไป ([อาจมีกรณีเพิ่มเติม](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).

Reviewer: ```suggestion
บัญชีที่ถูก restricted จะไม่สามารถเล่นออนไลน์ได้ ทำคะแนนใหม่ พูดคุยกับผู้เล่นอื่น ซึ่งก็คือการ "แบน" การทำอะไรก็ตามที่เกี่ยวกับชุมชน การ restict นั้นสามารถนำออกได้โดยการ appeal กับทีมงานของ osu! หลังจาก 3 เดือนได้ผ่านไป ([อาจมีกรณีเพิ่มเติม](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).
```
Make capitalization more consistent

> บัญชีที่ถูก Restricted จะไม่สามารถเล่นออนไลน์ได้, ทำคะแนนใหม่, พูดคุยกับผู้เล่นอื่น ซึ่งก็คือการ "แบน" การทำอะไรก็ตามที่เกี่ยวกับชุมชน การ restict นั้นสามารถนำออกได้โดยการ appeal กับทีมงานของ osu! หลังจาก 3 เดือนได้ผ่านไป ([อาจมีกรณีเพิ่มเติม](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).

Reviewer: ```suggestion
บัญชีที่ถูก Restricted จะไม่สามารถเล่นออนไลน์, ทำคะแนนใหม่, พูดคุยกับผู้เล่นอื่น ซึ่งก็คือการ "แบน" การทำอะไรก็ตามที่เกี่ยวกับชุมชน การ restict นั้นสามารถนำออกได้โดยการส่งคำร้องขอถึงทีมงานของ osu! หลังจาก 3 เดือนได้ผ่านไป ([อาจมีกรณีเพิ่มเติม](/wiki/Help_Centre/Account_Restrictions#common-restriction-reasons-and-cooldowns)).
```

> การละเมิดที่รุนแรงมาก ๆ จนเกินกฏของพวกเรา และเป็นสิ่งที่เกี่ยวข้องกับสิ่งผิดกฏหมาย จะทำให้บัญชีถูกลบออกจากระบบทันที ทั้งนี้รวมถึง ภาพที่แสดงความรุนแรงมาก ๆ หรืออะไรที่ใกล้เคียง เช่น สื่อลามกเด็ก หรือสื่อการละเมิดอื่น ๆ

Reviewer: ```suggestion
การละเมิดที่รุนแรงมาก ๆ จนเกินกฎของพวกเรา และเป็นสิ่งที่เกี่ยวข้องกับสิ่งผิดกฎหมาย จะทำให้บัญชีถูกลบออกจากระบบทันที ทั้งนี้รวมถึง ภาพที่แสดงความรุนแรงมาก ๆ หรืออะไรที่ใกล้เคียง เช่น สื่อลามกเด็ก หรือสื่อการละเมิดอื่น ๆ
```

> การทำผิดกฏที่รุนแรงมาก ๆ จนเกินกฏของพวกเรา และเป็นสิ่งที่เกี่ยวข้องกับสิ่งผิดกฏหมาย จะทำให้บัญชีถูกลบออกจากระบบทันที ทั้งนี้รวมถึง ภาพที่แสดงความรุนแรงมาก ๆ หรืออะไรที่ใกล้เคียง เช่น สื่อลามกเด็ก และรวมถึงการละเมิดต่าง ๆ

Reviewer: ```suggestion
การละเมิดที่รุนแรงมาก ๆ จนเกินกฏของพวกเรา และเป็นสิ่งที่เกี่ยวข้องกับสิ่งผิดกฏหมาย จะทำให้บัญชีถูกลบออกจากระบบทันที ทั้งนี้รวมถึง ภาพที่แสดงความรุนแรงมาก ๆ หรืออะไรที่ใกล้เคียง เช่น สื่อลามกเด็ก หรือสื่อการละเมิดอื่น ๆ
```
offense แปลใหม่ ละก็แปลหลังสุด reword
